examples/custom_converters/elu_converter/setup.py

```diff
@@ -0,0 +1,24 @@
+import os
+from setuptools import setup, Extension
+from torch.utils import cpp_extension
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+# library_dirs should point to the libtrtorch.so, include_dirs should point to the dir that include the headers
+# 1) download the latest package from https://github.com/NVIDIA/TRTorch/releases/
+# 2) Extract the file from downloaded package, we will get the "trtorch" directory
+# 3) Set trtorch_path to that directory
+trtorch_path = os.path.abspath("/home/bowa/Downloads/trtorch")

```

Comment:
We should set this to some invalid string so people have to change it, or have the code check to see its changed from default instead of your particular downloads dir

Reply:
sorry I missed that, I think it's ok now